app/src/main/java/com/fstyle/androidtrainning/MainActivity.java

```diff
@@ -2,12 +2,36 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
 
 public class MainActivity extends AppCompatActivity {
 
+    private EditText edtSo1, edtSo2, edtKetQua;
+    private Button btCong;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        initViews();
+        btCong.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                int so1 = Integer.parseInt(edtSo1.getText().toString());
+                int so2 = Integer.parseInt(edtSo2.getText().toString());
+                int tong = so1 + so2;
+                edtKetQua.setText("" + tong);
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.